src/components/Tabs/TabItem.tsx

```diff
@@ -0,0 +1,22 @@
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Tabs/tabs';
+import { PropsWithChildren } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+interface ITabItemProps {
+  url: string;
+  index?: number;
+}
+
+export const TabItem = ({ children, url, index }: PropsWithChildren<ITabItemProps>) => {
+  const navigate = useNavigate();
+  const { pathname } = useLocation();
+
+  return (
+    <li className={css(styles.tabsItem, pathname.includes(url) && styles.modifiers.current)}>
```

Comment:
check if there is `key` missing warning in developer console, see https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
There is no such error since all ```TabItem```s are declared manually.